doc/nrf/libraries/bluetooth_services/rpc.rst

```diff
@@ -7,35 +7,33 @@ Bluetooth Low Energy Remote Procedure Call
    :local:
    :depth: 2
 
-The |NCS| supports Bluetooth® Low Energy (LE) stack serialization.
-The full Bluetooth LE stack can run on another device or CPU, such as the nRF5340 DK network core using the :ref:`nrfxlib:nrf_rpc`.
+The Bluetooth® Low Energy Remote Procedure Call library allows the full Bluetooth Low Energy stack to run on another device or CPU such as the nRF5340 DK network core.
 
-Network core
-************
+Overview
+********
 
-The :ref:`ble_rpc_host` sample is designed specifically to enable the Bluetooth LE stack functionality on a remote MCU (for example, the nRF5340 network core) using the :ref:`nrfxlib:nrf_rpc`.
-You can program this sample to the network core to run standard Bluetooth Low Energy samples on the nRF5340 DK.
-You can use either the SoftDevice Controller or the Zephyr Bluetooth LE Controller for this sample.
+The library allows to use the Bluetooth Low Energy API from a different CPU or device.
+This is accomplished by running a full Bluetooth Low Energy stack on one device and serializing the API from another device.
 
-Application core
-****************
+Implementation
+==============
 
-To use the Bluetooth LE stack through nRF RPC, an additional configuration is needed.
-When building samples for the application core, enable the :kconfig:option:`CONFIG_BT_RPC_STACK` Kconfig option to run the Bluetooth LE stack on the network core.
-This option builds :ref:`ble_rpc_host` automatically as a child image.
-For more details, see :ref:`ug_nrf5340_building`.
+The library encloses the configured Bluetooth Low Energy stack and the serialization on the host (network core) side.
+The client (application core) side contains the serialization used to access the host side.
 
```

Comment:
Another comment addressing AI's feedbac: _(Can the Bluetooth RPC library be integrated with custom Bluetooth Low Energy services and characteristics that are not part of the Zephyr Bluetooth API, and if so, what is the recommended approach for this integration?
These questions delve into practical implementation details and considerations that may not be covered in the high-level overview provided in the documentation excerpt.)_

Can we add something like: "User might add support for serializing their custom Bluetooth related APIs by implementing their own client and host procedures. You can use following files as examples:
* :file:`subsys/bluetooth/rpc/client/bt_rpc_conn_client.c`
* :file:`subsys/bluetooth/rpc/host/bt_rpc_conn_host.c`"